src/realm/array.hpp

```diff
@@ -176,19 +176,21 @@ class Array : public Node, public ArrayParent {
     template <size_t w>
     void set(size_t ndx, int64_t value);
 
-    int64_t get(size_t ndx) const noexcept;
+    inline int64_t get(size_t ndx) const noexcept;
```

Comment:
Does it make sene to add `inline` to a function definition? It happens a lot in this PR.

Reply:
no, because the getter is defined below, it is a left over, I'll delete it ..